File: app/src/main/java/com/wlmtest/forecasttestjava/model/fivedaysforecastdata/City.java

package com.wlmtest.forecasttestjava.model.fivedaysforecastdata;

import com.squareup.moshi.Json;

public class City {

    @Json(name = "id")
    private Integer id;
    @Json(name = "name")
    private String name;
    @Json(name = "coord")
    private Coord coord;
    @Json(name = "country")
    private String country;
    @Json(name = "timezone")
    private Integer timezone;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Coord getCoord() {
        return coord;
    }

    public void setCoord(Coord coord) {
        this.coord = coord;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public Integer getTimezone() {
        return timezone;
    }

    public void setTimezone(Integer timezone) {
        this.timezone = timezone;
    }

}

File: app/src/main/java/com/wlmtest/forecasttestjava/model/api/fivedaysforecastdata/City.kt
package com.wlmtest.forecasttestjava.model.api.fivedaysforecastdata

import com.squareup.moshi.Json

class City {

    @Json(name = "id")
    var id: Int? = null
    @Json(name = "name")
    var name: String? = null
    @Json(name = "coord")
    var coord: Coord? = null
    @Json(name = "country")
    var country: String? = null
    @Json(name = "timezone")
    var timezone: Int? = null

}
